src/components/App.js

```diff
@@ -1,27 +1,10 @@
 import React from 'react';
-import logo from '../res/logo.svg';
+import AdminPage from './AdminPage/AdminPage';
 
 function App() {
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit
-          <code>
-            src/App.js
-          </code>
-          and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+      <AdminPage />
```

Comment:
This way whenever the app loads it will open the admin page. We don't want that. Your branch is not up to date with master. The master branch has a component called [`Router`](https://github.com/mai-youth/website-fe/blob/master/src/Router.js) that loads components based on the url. Use that instead.

Check PR https://github.com/mai-youth/website-fe/pull/15 for more details.

Reply:
To be clear, you should not change `App` at all. Just update the router if you need to.

Reply:
I will pull down the changes in master and use the router. I only added the Admin Page to app to see what it looked like i was meant to remove it from there.